libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -141,6 +144,13 @@ const AP_Param::GroupInfo AP_Follow::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("_OPTIONS", 11, AP_Follow, _options, 0),
 
+    // @Param: _TIMEOUT
+    // @DisplayName: Follow timeout
+    // @Description: Follow position update from lead - timeout after x milliseconds
```

Comment:
might as well go to seconds?

Reply:
Copter (Rover?) uses a hardcoded timeout of 600ms for this value.

Reply:
I'd prefer seconds too